dashboards-and-dashboard-groups/CICD-Pipeline/CICD_pipeline.tf

```diff
@@ -0,0 +1,476 @@
+provider "signalfx" {
+  api_url    = "https://api.${var.realm}.signalfx.com"
+  auth_token = var.access_token
+}
+
+# signalfx_event_feed_chart.CICD_pipeline_0:
+resource "signalfx_event_feed_chart" "CICD_pipeline_0" {
+  name         = "Event Feed"
+  program_text = <<-EOF
+        C = events(eventType='Automated Rollback initiated').publish(label='C')
+        B = events(eventType='${var.push_event_name}').publish(label='B')
+        A = alerts(detector_name='CI/CD Service Outlier').publish(label='A')
```

Comment:
Might be worth calling out these static named events/alerts in the `README.md` and how they are useful. 

Might also consider making them changable as a variable so users can add their own "automated roll back" events and the detector name they want?